sql/geometry.go

```diff
@@ -70,10 +76,10 @@ const (
 	WKBPointID
 	WKBLineID
 	WKBPolyID
-	WKBMultiPointID
-	WKBMultiLineID
-	WKBMultiPolyID
-	WKBGeoCollectionID
+	WKBMPointID
+	WKBMLineID
+	WKBMPolyID
+	WKBGeomCollectionID
```

Comment:
these are better as is, it's hard to visually differentiate between `WKBPointId` and `WKBMPointId`